tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ManagedIdentityTests.cs

```diff
@@ -1176,23 +1177,22 @@ public async Task UnsupportedManagedIdentitySource_ThrowsExceptionDuringTokenAcq
             string UnsupportedEndpoint = "unsupported://endpoint";
 
             using (new EnvVariableContext())
+            using (var httpManager = new MockHttpManager())
```

Comment:
simply adding this line fixes the issue for me

```
// Disabling shared cache options to avoid cross test pollution.
miBuilder.Config.AccessorOptions = null;
```

Reply:
That won't fix the issue though because we need a mocked probe failure.

Reply:
Also, Bogdan removed all instances of that with his `ResetStateForTest` PR